lib/active_admin/filters/resource_extension.rb

```diff
@@ -109,19 +109,11 @@ def filter_lookup
       def default_filters
         result = []
         result.concat default_association_filters if namespace.include_default_association_filters
-        result.concat default_content_filters
+        result.concat content_columns
         result.concat custom_ransack_filters
         result
       end
 
-      def custom_ransack_filters
-        if resource_class.respond_to?(:_ransackers)
-          resource_class._ransackers.keys.map(&:to_sym)
-        else
-          []
-        end
-      end
-
```

Comment:
This method was not removed, was just moved around. Any reason? Moving it back here would simplify the diff and make reviewing this easier.